README.md

```diff
@@ -51,25 +51,25 @@ Dive into understanding `fastapi-serve` through our comprehensive documentation
 - 🚀 **Getting Started**
     - 🧱 [Deploy a Simple FastAPI Application](docs/simple/)
     - 🖥️ [Dig deep into the `fastapi-serve` CLI](docs/CLI.md)
-    - ⚙️ [Understanding Configuration and Pricing](docs/CONFIG.MD)
+    - ⚙️ [Understanding Configuration and Pricing on Jina AI Cloud](docs/CONFIG.MD)
     - 🔄 [Upgrade your FastAPI applications with zero downtime](docs/upgrades/)
-    - 🏢 Managing Larger Applications with Complex Directory Structure (Example TBD!)
+    - 🏢 Managing Larger Applications with Complex Directory Structure (Coming Soon!)
```

Comment:
I would like to separate `Coming Soon` & `Example TBD` as the former denotes a new feature whereas the latter is for pending docs for existing features.

Reply:
"Example TBD" is not going to mean much to readers. How about if I say "Documentation in progress"?

Reply:
Sounds good